terraform/environments/ci/main.tf

```diff
@@ -0,0 +1,22 @@
+terraform {
+    backend "s3" {
+        bucket = "hashbang-terraform-ci"
+        key = "state.tfstate"
+        region = "us-west-2"
+        lock_table = "hashbang-terraform-ci"
+    }
+}
+
+provider "aws" {
+    region = "us-west-2"
+}
+
+module "vpc" {
+    source = "github.com/terraform-community-modules/tf_aws_vpc?ref=v1.0.6"
+    name = "ci"
+    cidr = "10.0.0.0/16"
```

Comment:
To further clarify, we can later (with a bit more $) upgrade this with the agents running in a dedicated autoscale group as dumb on-demand workers of which they will churn IP addresses pretty quick when doing a bunch of builds/jobs at once.